RELNOTES.md

```diff
@@ -2,6 +2,81 @@
 
 © Copyright EnterpriseDB UK Limited 2015-2023 - All rights reserved.
 
+## v23.25 (unreleased)
+
+### Notable changes
+
+- TPA-445 Support upgrades from BDR 3.7 to PGD5
+
+  A BDR 3.7 cluster created with BDR-Always-ON architecture, can now be
+  upgraded to the PGD-Always-ON architecture, running PGD5, by using
+  `tpexec reconfigure` to generate a revised config.yml for PGD5 and then
```

Comment:
```suggestion
  `tpaexec reconfigure` to generate a revised config.yml for PGD5 and then
```

Reply:
IMO it's a long paragraph. How about. "A BDR 3.7 cluster created with BDR-Always-ON architecture running PGD5 can now be upgraded to the PGD-Always-ON architecture. `tpexec reconfigure` will generate a revised config.yml..." sounds a bit easier to read.